src/parsers/parseArrayToStringWithAticle.ts

```diff
@@ -0,0 +1,17 @@
+const parseArrayToStringWithArticle = (arr: []): String => {
+  let str = "";
+  let arrSize = arr.length;
```

Comment:
Use `const` instead of` let`. This variable does not mutate during the flow of the function and `const` guarantees the immutability of primitive values.